apollo-router-core/src/query_planner/mod.rs

```diff
@@ -45,10 +45,10 @@ pub(crate) enum PlanNode {
 
 impl QueryPlan {
     /// Validate the entire request for variables and services used.
-    #[tracing::instrument(name = "validation", level = "debug")]
+    #[tracing::instrument(name = "validation", level = "debug", skip_all)]
     pub fn validate_request(
         &self,
-        request: &Request,
+        _request: &Request,
```

Comment:
the only use of that argument was by the tracing attribute. Do we assume that it will be used at some point in that function, or can I remove it?

Reply:
Maybe we can rename this to `validate_plan` or something? It seems like it validate a query plan and makes sure the plan will be able to be executed

Reply:
We can remove this and as @o0Ignition0o suggest, this is not validating the request but the plan itself.

It is possible it's a leftover of #227

Reply:
removed in c34d39c